packages/container/tests/test-helpers/build-owner.js

```diff
@@ -3,11 +3,39 @@ import Registry from 'container/registry';
 import RegistryProxy from 'ember-runtime/mixins/registry_proxy';
 import ContainerProxy from 'ember-runtime/mixins/container_proxy';
 
+// copied from ember-htmlbars/tests/integration/local-lookup-test.js
+function buildResolver() {
+  let resolver = {
+    resolve() { },
+    expandLocalLookup(fullName, sourceFullName) {
+      let [sourceType, sourceName ] = sourceFullName.split(':');
+      let [type, name ] = fullName.split(':');
+
+      if (type !== 'template' && sourceType === 'template' && sourceName.slice(0, 11) === 'components/') {
+        sourceName = sourceName.slice(11);
+      }
+
+      if (type === 'template' && sourceType === 'template' && name.slice(0, 11) === 'components/') {
+        name = name.slice(11);
+      }
+
+
+      let result = `${type}:${sourceName}/${name}`;
+
+      return result;
+    }
+  };
+
+  return resolver;
+}
+
 export default function buildOwner(props) {
   let Owner = EmberObject.extend(RegistryProxy, ContainerProxy, {
     init() {
       this._super(...arguments);
-      let registry = new Registry(this._registryOptions);
+      let registry = new Registry(this._registryOptions || {
+        resolver: buildResolver()
```

Comment:
I don't think that all tests should get a local lookup enabled resolver.  Especially since the actual patterns for how local lookup will work are still up in the air (see https://github.com/emberjs/rfcs/pull/124).   It seems better (to me) to inline this change, and the resolver above in the local-lookup-test file until those things are settled...